mainnet/jobs.json

```diff
@@ -1,202 +1,218 @@
-    "power": 5,
-    "selectorType": 0,
-    "name": "usdcusd_gemini",
-    "selector": "last",
-    "url": "https://api.gemini.com/v1/pubticker/usdcusd"
-  },
-  {
-    "weight": 1,
-    "power": 5,
-    "selectorType": 0,
-    "name": "usdcusd_coinbase",
-    "selector": "data.amount",
-    "url": "https://api.coinbase.com/v2/prices/USDC-USD/spot"
-  },
-  {
-    "weight": 1,
-    "power": 5,
-    "selectorType": 0,
-    "name": "usdcusd_kraken",
-    "selector": "result.USDCUSD.c[0]",
-    "url": "https://api.kraken.com/0/public/Ticker?pair=USDCUSD"
-  },
-  {
-    "weight": 1,
-    "power": 5,
-    "selectorType": 0,
-    "name": "usdcusd_bitstamp",
-    "selector": "last",
-    "url": "https://www.bitstamp.net/api/v2/ticker/usdcusd/"
-  },
-  {
-    "weight": 1,
-    "power": 5,
-    "selectorType": 0,
-    "name": "usdtusd_gemini",
-    "selector": "last",
-    "url": "https://api.gemini.com/v1/pubticker/usdtusd"
-  },
-  {
-    "weight": 1,
-    "power": 5,
-    "selectorType": 0,
-    "name": "usdtusd_coinbase",
-    "selector": "data.amount",
-    "url": "https://api.coinbase.com/v2/prices/USDT-USD/spot"
-  },
-  {
-    "weight": 1,
-    "power": 5,
-    "selectorType": 0,
-    "name": "usdtusd_kraken",
-    "selector": "result.USDTZUSD.c[0]",
-    "url": "https://api.kraken.com/0/public/Ticker?pair=USDTZUSD"
-  },
-  {
-  "weight": 1,
-  "power": 5,
-  "selectorType": 0,
-  "name": "usdtusd_stablecoin_stats",
-  "selector": "USDT.kraken.USDT_USD.price",
-  "url": "https://stablecoinstats.com/api/tickers"
-  }
-]
+    {
+      "weight": 1,
+      "power": 2,
+      "selectorType": 0,
+      "name": "ethusd_gemini",
+      "selector": "last",
+      "url": "https://api.gemini.com/v1/pubticker/ethusd"
+    },
+    {
+      "weight": 1,
+      "power": 2,
+      "selectorType": 0,
+      "name": "ethusd_kucoin",
+      "selector": "data.ETH",
+      "url": "https://api.kucoin.com/api/v1/prices?base=USD&currencies=ETH"
+    },
+    {
+      "weight": 1,
+      "power": 2,
+      "selectorType": 0,
+      "name": "ethusd_kraken",
+      "selector": "result.XETHZUSD.c[0]",
+      "url": "https://api.kraken.com/0/public/Ticker?pair=ETHUSD"
+    },
+    {
+      "weight": 1,
+      "power": 2,
+      "selectorType": 0,
+      "name": "ethusd_coinbase",
+      "selector": "data.amount",
+      "url": "https://api.coinbase.com/v2/prices/ETH-USD/spot"
+    },
+    {
+      "weight": 1,
+      "power": 2,
+      "selectorType": 0,
+      "name": "ethusd_coingecko_api",
+      "selector": "ethereum.usd",
+      "url": "https://api.coingecko.com/api/v3/simple/price?ids=ethereum&vs_currencies=usd"
+    },
+    {
+      "weight": 1,
+      "power": 2,
+      "selectorType": 0,
+      "name": "ethusd_bitfinex",
+      "selector": "last_price",
+      "url": "https://api.bitfinex.com/v1/pubticker/ethusd"
+    },
+    {
+      "weight": 1,
+      "power": 2,
+      "selectorType": 0,
+      "name": "ethusd_cryptocompare",
+      "selector": "[USD]",
```

Comment:
The selector here should be `USD` instead of `[USD]` for `cryptocompare` API's, our current schain mainnet has this change @yohanelly95